tntconcept-binnacle/src/main/kotlin/com/autentia/tnt/binnacle/services/UserService.kt

```diff
@@ -27,8 +26,9 @@ class UserService internal constructor(
     fun findAllActive(): List<User> =
         userRepository.find()
 
+    fun getByUserName(userName: String): User =
+        userRepository.findByUsername(userName) ?: error("There isn't authenticated user")
```

Comment:
El mensaje de error quizás puede confundir.  Sería mejor un mensaje con `user not found` como el otro método